src/mlpa/core/pg_services/pg_service.py

```diff
@@ -12,10 +11,19 @@ class PGService:
 
     def __init__(self, db_name: str):
         self.db_name = db_name
-        self.db_url = os.path.join(env.PG_DB_URL, db_name)
+        self.db_url = f"{env.PG_DB_URL.rstrip('/')}/{db_name}"
         self.connected = False
         self.pg = None
 
+    async def _get_prepared_statement(self, conn: asyncpg.Connection, query: str):
+        stmt_cache = getattr(conn, "_mlpa_stmt_cache", None)
+        if stmt_cache is None:
+            stmt_cache = {}
+            conn._mlpa_stmt_cache = stmt_cache
+        if query not in stmt_cache:
+            stmt_cache[query] = await conn.prepare(query)
+        return stmt_cache[query]
```

Comment:
Won't this blow up the memory of our pods? Each request is stored until pod restart right? Is there a `max_store` variable we could add? Maybe store for up to 1 day? What do you think?